packages/render-fidelity-tools/test/renderers/gltfpt/shader/material.glsl

```diff
@@ -0,0 +1,182 @@
+/* @license

```

Comment:
not sure where these shader files come from. It's not in the master branch, but they are generated after i build(not even sure if they comes from building). Should i commit them?

Reply:
those are left overs from the renaming I did to the pathtracing renderer (gltfpt -> dspbr-pt). the shaders getting copied over from the node module during `prepare`. gltfpt should not exist anymore and the shaders are safe to delete.

Reply:
Thanks for the explanation!